src/pages/settings/AddSecondaryLoginPage.js

```diff
@@ -107,20 +96,26 @@ class AddSecondaryLoginPage extends Component {
                     onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_PROFILE)}
                     onCloseButtonPress={() => Navigation.dismissModal()}
                 />
-                <ScrollView style={styles.flex1} contentContainerStyle={styles.p5}>
+                <Form
+                    formID={ONYXKEYS.FORMS.SECONDARY_LOGIN_FORM}
+                    validate={this.validate}
+                    onSubmit={this.onSubmit}
+                    submitButtonText={this.props.translate('addSecondaryLoginPage.sendValidation')}
+                    style={[styles.flexGrow1, styles.mh5]}
+                >
                     <Text style={[styles.mb6]}>
                         {this.props.translate(this.formType === CONST.LOGIN_TYPE.PHONE
                             ? 'addSecondaryLoginPage.enterPreferredPhoneNumberToSendValidationLink'
                             : 'addSecondaryLoginPage.enterPreferredEmailToSendValidationLink')}
                     </Text>
                     <View style={styles.mb6}>
                         <TextInput
+                            autoFocus
+                            shouldDelayFocus
```

Comment:
Since `shouldDelayFocus` is only needed for android, we should use platform specific files for this variable so that we only delay the focus at android.

Reply:
Yes, I am planning to do this. I am waiting for https://github.com/Expensify/App/pull/12513